org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/model/ResourceForIEditorInputFactory.java

```diff
@@ -107,6 +112,8 @@ protected Resource createResource(IStorage storage) throws CoreException {
 	}
 
 	protected Resource createResourceFor(IStorage storage) throws CoreException {
+		if (storage.getFullPath() == null) 
```

Comment:
I wouldn't expect a createXYZ method to return null.

Reply:
I changed it.

Reply:
Thank you.